project/launchers.sc

```diff
@@ -79,10 +79,11 @@ trait Launchers extends CsModule {
         if (usesDocker) s"/data/$staticLibDirName"
         else staticLibDir().path.toString
       val extraOpts =
-        if (Properties.isLinux)
+        if (Properties.isLinux && arch == "aarch64")
           Seq(
             // required on the Linux / ARM64 CI in particular (not sure why)
-            "-Djdk.lang.Process.launchMechanism=vfork" // https://mbien.dev/blog/entry/custom-java-runtimes-with-jlink
+            "-Djdk.lang.Process.launchMechanism=vfork", // https://mbien.dev/blog/entry/custom-java-runtimes-with-jlink
```

Comment:
@alexarchambault Based on this comment, I think it should be safe to just have this property set for the aarch64 builds? You introduced that in #2239. I think we should see CI complaining if the non-aarch64 Linux binaries need to be build this flag way as well, hopefully (?)

Reply:
Yeah, that must have been an oversight. Non-aarch64 binaries are [tested on PRs too](https://github.com/coursier/coursier/blob/f730535d8d9c8bdb885d353c5665a6785510cf7f/.github/workflows/native-tests.yml), so if the CI doesn't complain, that's fine.